e2e/cypress/tests/support/ldap_server_commands.ts

```diff
@@ -1,18 +1,33 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {ChainableT} from 'tests/types';
+
 import {getRandomId} from '../utils';
 
 const ldapTmpFolder = 'ldap_tmp';
```

Comment:
i think we can revert this as dont need to change anything in this file once we merge the latest master.

Reply:
reverted